portfolio/src/main/java/com/google/sps/data/Comments.java

```diff
@@ -0,0 +1,141 @@
+package com.google.sps.data;
+
+import com.google.appengine.api.datastore.*;
+import com.google.appengine.api.datastore.Query;
+import java.util.*;
+
+/** Represents a datastore of comments.
+ * @author Olga Shimanskaia
+ * @author oshimanskaia@google.com
```

Comment:
The `@author` line is optional, but feel free to keep it. However, multiple author lines usually means multiple authors. A few alternatives:

- Olga Shimanskaia
- Olga Shimanskaia \<oshimanskaia@google.com\> (remember to change that when your account gets deactivated, to make sure people seeing this project use the correct email to reach out to you)
- Olga Shimanskaia \<a personal email of your choice\>
- jeinygroove
- oshimanskaia
- your personal email

Note: I personally don't see emails as author tags usually.

Same for other files